src/components/ImportGithub.js

```diff
@@ -0,0 +1,60 @@
+import { Fragment, useState } from 'react'
+import { div, h, h2 } from 'react-hyperscript-helpers'
+import { ButtonPrimary } from 'src/components/common'
+import { TextInput } from 'src/components/input'
+import { submitMethod } from 'src/components/method-common'
+import { withBusyState } from 'src/libs/utils'
+
+// { tooltip: 'Link must start with https://github.com or https://raw.githubusercontent.com' },
```

Comment:
Should we add this is as a validation on workflow url field?

Reply:
I meant to pose the question to you and Chris. I think it could be useful to have in case there's a question about what a proper link looks like.

Reply:
I think that's a good idea. We can add this tooltip. But I also think we should validate the url entered before sending it to CBAS. Validating in the UI will help avoid unnecessary trips to the API just for the import to fail. If the url is invalid the user will be able to edit the url in UI itself.